docs/conceptual/array.contains['t]-function-[fsharp].md

```diff
@@ -0,0 +1,75 @@
+---
+title: Array.contains<'T> Function (F#)
+description: Array.contains<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Array.contains<'T> Function (F#)
+
+Evaluates to `true` if the given element is in the given array.
```

Comment:
Perfect … it happens a little later on the page too.

From: Libo Zeng [mailto:notifications@github.com]
Sent: Saturday, June 25, 2016 4:44 AM
To: Microsoft/visualfsharpdocs <visualfsharpdocs@noreply.github.com>
Cc: Kevin Ransom <Kevin.Ransom@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Microsoft/visualfsharpdocs] List.contains and Array.contains (#187)


In docs/conceptual/array.contains['t]-function-[fsharp].md<https://github.com/Microsoft/visualfsharpdocs/pull/187#discussion_r68487878>:

> @@ -0,0 +1,75 @@

> +---

> +title: Array.contains<'T> Function (F#)

> +description: Array.contains<'T> Function (F#)

> +keywords: visual f#, f#, functional programming

> +author: liboz

> +---

> +

> +# Array.contains<'T> Function (F#)

> +

> +Evaluates to `true` if the given element is in the given array.

How about something like this: Evaluates to true if the given element is in the input array.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/visualfsharpdocs/pull/187/files/8f591c92a19d68d25b39625fe7b1eceeb09d149c#r68487878>, or mute the thread<https://github.com/notifications/unsubscribe/AE76FkJhhPv_KwRAzv_wIoNxyAc9aldhks5qPRSdgaJpZM4I-PnL>.